modules/installation-osp-enabling-kuryr.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// *  installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-enabling-kuryr_{context}"]
+= Enabling Kuryr
+
+You can enable Kuryr in your project by editing a parameter in `install-config.yaml`.
+
+.Prerequisites
+
+* Kuryr requires greater than the minimum recommended resources. As an administrator, increase OpenStack's resource quotas:
++
+----
+$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 --subnets 100 --networks 100 <project>
+----
+
+* Enable the `trunks` extension on your OpenStack project
+
+* Install and configure Octavia components in the OpenStack environment
+** Depending on your OpenStack environment, Octavia may not support UDP listeners. If it doesn't, your deployment will not support UDP services.
+
+* If the default ML2/OVS Neutron driver is used, set the firewall to `openvswitch` intead of `ovs_hybrid`
+
+* Have a valid `install-config.yaml` file
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN:
+
+.Procedure
+
+. Modify the `install-config.yaml` file to include `Kuryr` as the desired `networking.networkType`:
+====
```

Comment:
Don't enclose code blocks with ====
You must add a + on a blank line to nest the codeblock properly.